packages/checkout/widgets-lib/src/context/handover-context/HandoverProvider.tsx

```diff
@@ -1,8 +1,6 @@
+/* eslint-disable no-console */
```

Comment:
probably a stale rule, if needed add to the line with the console log exception?